common/src/main/java/io/homeassistant/companion/android/common/data/integration/impl/IntegrationRepositoryImpl.kt

```diff
@@ -224,6 +224,7 @@ class IntegrationRepositoryImpl @Inject constructor(
                     ).isSuccessful
             } catch (e: Exception) {
                 // Ignore failure until we are out of URLS to try!
+                Log.d(TAG, "Unable to send tag data for URL: $it with data: $data", e)
```

Comment:
Not sure if we want to add this, it's going to spam a lot of logs for some peoples setups.

Reply:
ok I was thinking maybe the call here failed.  I will instead add a logger to `handleTag` function to determine if the call was successful or not.  I suspect based on the logging, the issue either happens when we are parsing the tag for the ID or when we send the data back home.